### PR TITLE
imagelib library: toward image manipulation independant from C library

### PR DESCRIPTION
**Pull request by craff**

### CODE REVIEW

File: packages/imagelib/imagelib.20160413/descr [hunk 1]
```diff
@@ -0,0 +1,17 @@
+The imagelib library implements image formats such as PNG or PPM
+
+The imagelib library implements image formats such as PNG or PPM  in
+OCaml, relying on only one external dependency: camlzip. However, we
+plan to reimplement zlib in OCaml at some point.
```
Comment (samoht) [inline packages/imagelib/imagelib.20160413/descr]: Have you looked at https://github.com/oklm-wsh/Decompress ? It's a pure OCaml implementation of zlib

File: packages/imagelib/imagelib.20160413/opam [hunk 2]
```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "imagelib"
```
Comment (samoht) [inline packages/imagelib/imagelib.20160413/opam]: Can you remove the `name` and `version` of that file? They are implicit in the directory name already. Thanks!

File: packages/imagelib/imagelib.20160413/descr (hunk 1 above)
Comment (craff) [inline packages/imagelib/imagelib.20160413/descr]: Thanks ... this will be done

Comment (hannesm) [inline packages/imagelib/imagelib.20160413/descr]: it is in opam https://opam.ocaml.org/packages/decompress/decompress.0.4/ :)

**Comment by avsm** [comment]
Pretty clean bulk build; https://github.com/avsm/opam-pr-build-results/blob/6230/README.md
(just oraclelinux missing from zlib depexts)

**Pull request merged**